python/test/unit/io/test_vtk.py

```diff
@@ -43,12 +46,12 @@ def test_save_1d_meshfunctions(tempfile, mesh_function_types, file_options,
     for d in range(mesh.topology.dim + 1):
         for t in mesh_function_types:
             mf = MeshFunction(t, mesh, mesh.topology.dim - d, type_conv[t](1))
-            VTKFile(tempfile + "mf.pvd", "ascii").write(mf)
-            f = VTKFile(tempfile + "mf.pvd", "ascii")
+            VTKFile(tempfile + "mf.pvd").write(mf)
+            f = VTKFile(tempfile + "mf.pvd")
             f.write(mf, 0.)
             f.write(mf, 1.)
-            for file_option in file_options:
-                VTKFile(tempfile + "mf.pvd", file_option).write(mf)
+            # for file_option in file_options:
+            #     VTKFile(tempfile + "mf.pvd", file_option).write(mf)
```

Comment:
Leftover commented code?

Reply:
Can remove it, its basically due to the fact that the VTK signature has changed (see all the xfail tests further down)

Reply:
Removed

Reply:
Removed.